android/library/build.gradle

```diff
@@ -38,19 +48,37 @@ android {
             matchingFallbacks = ['debug', 'release']
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.polidea.rxandroidble:rxandroidble:1.7.1"
-    testImplementation 'junit:junit:4.12'
+    api 'io.reactivex.rxjava2:rxjava:2.2.17'
+    api "com.polidea.rxandroidble2:rxandroidble:1.17.2"
 }
 
-task sourcesJar(type: Jar) {
-    classifier = 'sources'
-    from android.sourceSets.main.java.sourceFiles
-}
+tasks.withType(PublishToMavenRepository).configureEach { it.enabled = false }
+
+publishing {
+    publications {
+        release(MavenPublication) {
+            groupId 'com.polidea.multiplatformbleadapter'
+            artifactId 'library'
+            version '2.0'
```

Comment:
should probably bump version aye? Or are we planning to do that once the release is ready?